engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -0,0 +1,65 @@
+package cromwell.services
+
+import akka.actor.Actor
+import cromwell.engine.workflow.WorkflowManagerActor.WorkflowMetadata
+import cromwell.engine.{ExecutionEventEntry, WorkflowId}
+import cromwell.engine.backend.JobKey
+import cromwell.services.MetadataServiceActor._
+import cromwell.webservice.WorkflowMetadataResponse
+import org.joda.time.DateTime
+
+object MetadataServiceActor {
+
+  sealed trait MetadataServiceActorMessage
+  case class SetStartTime(jobKey: JobKey, startTime: DateTime)
+  case class SetEndTime(jobKey: JobKey, time: DateTime)
+  case class AddExecutionEvent(jobKey: JobKey, executionEventEntry: ExecutionEventEntry)
+  /*
+  We'd have a "set" message for each of these:
```

Comment:
Can you give an example/suggestion what you'd like to see (specifically, like instead of the set)

> On Apr 5, 2016, at 7:38 PM, Jeff Gentry <notifications@github.com> wrote:
> 
> In engine/src/main/scala/cromwell/services/MetadataServiceActor.scala:
> 
> > +import akka.actor.Actor
> > +import cromwell.engine.workflow.WorkflowManagerActor.WorkflowMetadata
> > +import cromwell.engine.{ExecutionEventEntry, WorkflowId}
> > +import cromwell.engine.backend.JobKey
> > +import cromwell.services.MetadataServiceActor._
> > +import cromwell.webservice.WorkflowMetadataResponse
> > +import org.joda.time.DateTime
> > +
> > +object MetadataServiceActor {
> > +
> > +  sealed trait MetadataServiceActorMessage
> > +  case class SetStartTime(jobKey: JobKey, startTime: DateTime)
> > +  case class SetEndTime(jobKey: JobKey, time: DateTime)
> > +  case class AddExecutionEvent(jobKey: JobKey, executionEventEntry: ExecutionEventEntry)
> > +  /*
> > +  We'd have a "set" message for each of these:
> Very strong +1
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>